js/crex24.js

```diff
@@ -1109,31 +1114,68 @@ module.exports = class crex24 extends Exchange {
     }
 
     async cancelOrder (id, symbol = undefined, params = {}) {
+        const response = await this.cancelOrders ([ id ], symbol, params);
+        return this.safeValue (response, 0);
+    }
+
+    async cancelOrders (ids, symbol = undefined, params = {}) {
+        if (!Array.isArray (ids)) {
+            throw new ArgumentsRequired (this.id + ' cancelOrders ids argument should be an array');
+        }
         await this.loadMarkets ();
         const request = {
-            'ids': [
-                parseInt (id),
-            ],
+            'ids': [],
         };
+        for (let i = 0; i < ids.length; i++) {
+            const id = parseInt (ids[i]);
+            request['ids'].push (id);
+        }
         const response = await this.tradingPostCancelOrdersById (this.extend (request, params));
         //
         //     [
         //         465448358,
         //         468364313
         //     ]
         //
-        return this.parseOrder (response);
+        return this.parseOrders (response);
     }
 
     async cancelAllOrders (symbol = undefined, params = {}) {
-        const response = await this.tradingPostCancelAllOrders (params);
-        //
-        //     [
-        //         465448358,
-        //         468364313
-        //     ]
-        //
-        return response;
+        let response = undefined;
+        let market = undefined;
+        const isSymbolArray = Array.isArray (symbol);
+        if (symbol === undefined) {
+            response = await this.tradingPostCancelAllOrders (params);
+            //
+            //     [
+            //         465448358,
+            //         468364313
+            //     ]
+            //
+        } else {
+            await this.loadMarkets ();
+            const request = {
+                'instruments': undefined,
+            };
+            if (isSymbolArray) {
+                for (let i = 0; i < symbol.length; i++) {
+                    market = this.market (symbol);
+                    request['instruments'].push (market['id']);
+                }
+            } else {
+                market = this.market (symbol);
+                request['instruments'] = [ market['id'] ];
+            }
```

Comment:
Let's not expect an array of symbols in the `symbol` argument. We don't need `isSymbolArray` here.

Reply:
In docs, it states they accept multiple symbols array (`string[]`) : https://docs.crex24.com/trade-api/v2/#cancellation-by-instrument
So, I assumed we could leave users that functionality to be used, and let them pass array if they want. No?

Reply:
@ttodua i'm afraid, that would lead to false-expectations from the unified api with the other exchanges.

Reply:
On the other hand, we could keep it here, just for their convenience.